[2022-07-22 12:59:44,448] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-01T00:00:00+00:00 [queued]>
[2022-07-22 12:59:44,875] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-01T00:00:00+00:00 [queued]>
[2022-07-22 12:59:45,128] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:45,152] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 12:59:45,155] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:45,483] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-01 00:00:00+00:00
[2022-07-22 12:59:45,553] {standard_task_runner.py:52} INFO - Started process 737 to run task
[2022-07-22 12:59:45,610] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-01T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpv5dv7p1w', '--error-file', '/tmp/tmp6jhw2774']
[2022-07-22 12:59:45,618] {standard_task_runner.py:80} INFO - Job 23: Subtask t2
[2022-07-22 12:59:45,702] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 12:59:46,761] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-01T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 12:59:47,940] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-01T00:00:00+00:00
[2022-07-22 12:59:47,945] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 12:59:48,394] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220601T000000, start_date=20220722T125944, end_date=20220722T125948
[2022-07-22 12:59:48,630] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 12:59:50,702] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:17:55,098] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-01T00:00:00+00:00 [queued]>
[2022-07-25 09:17:55,269] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-01T00:00:00+00:00 [queued]>
[2022-07-25 09:17:55,276] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:55,282] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:17:55,283] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:55,439] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-01 00:00:00+00:00
[2022-07-25 09:17:55,453] {standard_task_runner.py:52} INFO - Started process 702 to run task
[2022-07-25 09:17:55,568] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-01T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpxp1kd8ck', '--error-file', '/tmp/tmp0ey10wu7']
[2022-07-25 09:17:55,607] {standard_task_runner.py:80} INFO - Job 21: Subtask t2
[2022-07-25 09:17:55,854] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:17:56,211] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-01T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:17:56,630] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-01T00:00:00+00:00
[2022-07-25 09:17:56,637] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:17:56,747] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220601T000000, start_date=20220725T091755, end_date=20220725T091756
[2022-07-25 09:17:56,849] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:17:56,989] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
